### DISCUSSION TITLE
How to use the project-template with an older Atoti version?

### DISCUSSION THREAD

**Discussion by amhatre** [atoti/atoti#792] [Q&A] [answered]
I would like to verify if a customer can run the [project-template](https://github.com/atoti/project-template). But the challenge is that they have Atoti 0.7.1.nightly, and it will take some time to upgrade to latest Atoti. 
Is there a way to run the project-template with 0.7.1? 

I removed the `poetry.lock`, set the atoti version to "0.7.1" and python version to ">=3.8,<3.9.7 || >3.9.7".

After that, command `poetry install` ran successfully. 
But when I run `poetry run app start` I get error - 
```
  File "C:\Projects\Lab\python\project-template\app\config.py", line 10, in <module>
    from pydantic import (
  File "C:\Projects\Lab\python\project-template\.venv\lib\site-packages\pydantic\__init__.py", line 206, in __getattr__
    return _getattr_migration(attr_name)
  File "C:\Projects\Lab\python\project-template\.venv\lib\site-packages\pydantic\_migration.py", line 279, in wrapper
    raise PydanticImportError(
pydantic.errors.PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.0.2/migration/#basesettings-has-moved-to-pydantic-settings for more details.
```

Since there will be breaking changes in the different releases, I don't think my changes above are sufficient. 

Is there an older branch that we can switch to?

**Comment by tibdex** [comment, marked as answer]
You can click on the **View blame prior to this change** icon [here](https://github.com/atoti/project-template/blame/5af7272d7a5bbded7fa263b586a914b370ee645f/poetry.lock#L5) until it's the Atoti version you want and checkout the corresponding commit.

**Reply by amhatre** [reply]
Thanks! This worked for me.
